Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Generic Jewelry Artifacts

### Changelog

- Added 21 types of amulets
- Added 21 types of rings
- 5 legendary, 10 epic, 15 rare, 20 uncommon and 25 common rings and amulets will be distributed among landed character who don't have an artifact at game start
- Added suffix+prefix naming system for jewelry

### How to test

- [Proofread this, pls](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit?usp=sharing "Spreadsheet")

- Check artifact spawn in-game
